subiquity/models/filesystem.py

```diff
@@ -1044,6 +1044,28 @@ def can_delete(self):
         return True
 
 
+@fsobj("zpool")
+class ZPool:
+    vdevs = attributes.reflist(backlink="_constructed_device")
+    pool = attr.ib()
```

Comment:
It would be nice if we could add type annotations to the fs model objects, I had to go to the docs to find out that "pool" here is a name.

Reply:
Done